### PR TITLE
[prometheus-cloudwatch-exporter] Allow for specifying automountserviceaccounttoken for prometheus-clou…

### PR DESCRIPTION
**Pull request by joshuastern**
#### What this PR does / why we need it:

Allow for specifying the automountServiceAccountToken field on the prometheus-cloudwatch-exporter service account.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist

- [X] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [X] Chart Version bumped
- [X] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

File: charts/prometheus-cloudwatch-exporter/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "0.10.0"
 description: A Helm chart for prometheus cloudwatch-exporter
 name: prometheus-cloudwatch-exporter
-version: 0.14.1
+version: 0.14.2
```
Comment (monotek) [inline charts/prometheus-cloudwatch-exporter/Chart.yaml:5]: New feature should raise minor version.

**Review by monotek — CHANGES_REQUESTED** [review]

File: charts/prometheus-cloudwatch-exporter/Chart.yaml (hunk 1 above)
Comment (joshuastern) [reply charts/prometheus-cloudwatch-exporter/Chart.yaml]: updated

**Review by monotek — DISMISSED** [review]

File: charts/prometheus-cloudwatch-exporter/Chart.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "0.10.0"
 description: A Helm chart for prometheus cloudwatch-exporter
 name: prometheus-cloudwatch-exporter
-version: 0.14.1
+version: 0.15.0
```
Comment (torstenwalter) [inline charts/prometheus-cloudwatch-exporter/Chart.yaml:5]: @joshuastern I just merged another PR could you bump the chart version. Your change looks good!

Comment (joshuastern) [reply charts/prometheus-cloudwatch-exporter/Chart.yaml]: bumped!

**Review by torstenwalter — APPROVED** [review]

**Review by torstenwalter — APPROVED** [review]

**Comment by torstenwalter** [comment]
@joshuastern Thank you!

**Pull request merged**